dev/refresh_source.py

```diff
@@ -434,7 +434,8 @@ def write_deck_run_script(self, repository):
 cd "$d"
 LOG_DIR=${{LOG_DIR:-../logs}}
 
-bash -c "(./start.sh >> '$LOG_DIR/{name}.log') 2>&1\
+npm install >& "$LOG_DIR/deck.log"
```

Comment:
If `start.sh` is just for use without refresh_source, why not just add some comments to it indicating that? Can't see the reason to make it more complicated. The nvm/npm/node stuff is already complicated enough.